backend/src/api/api_v1/endpoints/admin.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from src import crud
+from src.api import deps
+from src.core.config import settings
+from src.db.db import get_session
+from src.models.user import User, UserCreate, UserResponse, UserUpdate
+from src.utils import send_new_account_email
+
+router = APIRouter()
```

Comment:
you need to include this router in backend/src/api/api_v1/api.py with the prefix admin